docs/0.3/using_purchase_order.md

```diff
@@ -0,0 +1,327 @@
+
+6.  Using the `gridd-alpha` command line, create a new organization by
+    specifying a unique organization ID, and the organization's name.
+
+    ```
+    root@gridd-alpha:/# grid organization create \
+    farm "Fred's Farm"
+    ```
+
+   This command creates and submits a transaction to create a new Pike
+   organization with the data you supplied, as well as a new Pike agent
+   with the `admin` role and `active` status.
+
+   The transaction is signed with the agent's private key, as derived from the
+   base name specified by `GRID_DAEMON_KEY` (or with the `-k` option on the
+   command line). The agent's public key is used as the agent ID.
+
+   **Note**: This command doesn't display any output. Instead, check the log
+   messages (in the terminal window where you started the Grid Docker
+   environment) for the success or failure of this operation.
+
+#### Create a Purchase Order Admin Role
+
+In order to submit transactions using other Grid features the submitting
+agent
+must possess an active role with the appropriate permissions.
+
+{:start="7"}
+
+7. Verify that the agent's public key file exists in
+`~/.grid/keys/alpha-agent.pub`. If not, you must specify contents of the
+public key file in the next command.
+
+1. Create a role that has the appropriate permissions. The following command
+shows how to create a role called `product-admin` with permissions to create,
+update, and delete schemas and products.
+
+    ```
+    root@gridd-alpha:/# grid role create \
+    farm po-admin \
+    --description "purchase order admin permissions" \
+    --active \
+    --allowed-orgs gnrl \
+    --permissions "po::can-create-order,po::can-create-version,po::can-update-order,po::can-update-version"
+    ```
+
+1. Assign this role to the admin agent for this organization
+
+    ```
+    root@gridd-alpha:/# grid agent update \
+    farm $(cat ~/.grid/keys/alpha-agent.pub) \
+    --active \
+    --role admin \
+    --role po-admin
+    ```
+
+### Create a "Vendor" Organization
+
+{:start="10"}
+
+10. In the `gridd-beta` command line, create an organization to act as a
+    vendor. This organization will manage purchase orders on behalf of Fred's
+    Farm.
+
+    ```
+    root@gridd-beta:/# grid organization create \
+    gnrl "General Store"
+    ```
+
+#### Create a Purchase Order Manager Role
+
+Create a role that inherits the purchase order permissions from the `po-admin`
+permission belonging to Fred's Farm
+
+{:start="11"}
+
+11. Create a role that inherits the purchase order permissions from the `po-admin`
+    permission belonging to Fred's Farm
+
+    ```
+    root@gridd-beta:/# grid role create gnrl po-manager \
+    --active \
+    --permissions “po::can-create-po,po::can-create-version,po::can-update-po,po::can-update-version” \
+    --inherit-from “farm.po-admin
+    ```
+
+1.  Add the new role to the admin agent for General Store
+
+    ```
+    root@gridd-beta:/# grid agent update \
+    gnrl $(cat ~/.grid/keys/alpha-agent.pub) \
```

Comment:
If we're making this org in the beta container, the pub key should probs be `beta-agent.pub` and you should probably add a note like the one in (currently) step 7 since this is a different container

Reply:
good catch. Added/updated